DOC-808: Add known issues to 5.8 release notes (#1957)
tinymce/tinymce-docs: release-notes/release-notes58.md

**Outline**

> - [Unexpected behavior when moving the caret using the PageDown key](#unexpectedbehaviorwhenmovingthecaretusingthepagedownkey)

Reviewer: This needs updating.

- [Accessibility when using keyboard navigation to resize the editor](#accessibilitywhenusingkeyboardnavigationtoresizetheeditor)

> ### Unexpected behavior when moving the caret using the PageDown key

Reviewer: This issue applies to both the `PageDown` and `End` key so we'll need to include that.

> ### Unexpected behavior when moving the caret using the PageUp, PageDown, and End keys

Reviewer: Not sure where `PageUp` came from here? The issue only applies to `PageDown` and `End` either way, so we'll need that updated.

Author: Whoops

**Issue**: This issue affects moving the caret over links using the PageDown key. If the caret is placed before a link and the PageDown key is pressed, the caret will move inside the link rather than past it.

**Workaround**: There was no known workaround at the time of the release.

### Accessibility when using keyboard navigation to resize the editor

> **Issue**: The editor can now be resized using keyboard navigation, however there is currently no available ARIA role to indicate the resize state for a screen reader or other assistive technology to consume.

Reviewer: ```suggestion
**Issue**: The editor can now be resized using keyboard navigation, however there is currently no available ARIA attributes to indicate the resize state for a screen reader or other assistive technology to consume.
```
This will entail more than just a role, so this just updates that to reflect that.

**Outline**

- [Unexpected behavior when moving the caret using the PageDown and End keys](#unexpectedbehaviorwhenmovingthecaretusingthepagedownandendkeys)
- [Accessibility when using keyboard navigation to resize the editor](#accessibilitywhenusingkeyboardnavigationtoresizetheeditor)
- [Content pasted from Google Docs cannot be outdented](#contentpastedfromgoogledocscannotbeoutdented)

### Unexpected behavior when moving the caret using the PageDown and End keys

> **Issue**: This issue affects moving the caret over links using the PageDown or End keys. If the caret is placed adjacent to a link and the key is pressed, the caret will move inside the link rather than past it.

Reviewer 2: We generally **bold** the names of keyboard keys elsewhere in the docs.
```suggestion
**Issue**: This issue affects moving the caret over links using the **PageDown** or **End** keys. If the caret is placed adjacent to a link and the key is pressed, the caret will move inside the link rather than past it.
```

Reviewer: Don't we normally put them in code blocks, not make them bold?

Reviewer 3: Looking at the keyboard shortcuts page, backticks

Reviewer 3: ```suggestion
**Issue**: This issue affects moving the caret over links using the `Page Down` or `End` keys. If the caret is placed adjacent to a link and the key is pressed, the caret will move inside the link rather than past it.
```

**Workaround**: There was no known workaround at the time of the release.

### Accessibility when using keyboard navigation to resize the editor

> **Issue**: The editor can now be resized using keyboard navigation, however there is currently no available ARIA specifications to indicate the resize state for a screen reader or other assistive technology to consume.

Reviewer 2: ```suggestion
**Issue**: The editor can now be resized using keyboard navigation, however there is currently no available ARIA specification to indicate the resize state for screen readers or other assistive technology to consume.
```

**Workaround**: There was no known workaround at the time of the release. We are tracking the relevant [ARIA GitHub issue](https://github.com/w3c/aria/issues/432) for a way to support this.

### Content pasted from Google Docs cannot be outdented

> **Issue**: This issue affects content pasted in from Google Docs using the [PowerPaste plugin]({{site.baseurl}}/plugins/premium/powerpaste/). When content is pasted into the editor, that contains indentation, it then cannot be outdented using the available toolbar buttons.

Reviewer 2: ```suggestion
**Issue**: This issue affects content pasted in from Google Docs using the [PowerPaste plugin]({{site.baseurl}}/plugins/premium/powerpaste/). When content is pasted into the editor that contains indentation, it then cannot be outdented using the available toolbar buttons.
```

> **Workaround**: The only known workaround is to configure TinyMCE to use margins instead of padding for indentation. This can be enabled by configuring the [`indent_use_margin` option]({{site.baseurl}}/configure/content-formatting/#indent_use_margin) 

Reviewer 3: ```suggestion
**Workaround**: The only known workaround is to configure TinyMCE to use margins instead of padding for indentation. This can be enabled by configuring the [`indent_use_margin` option]({{site.baseurl}}/configure/content-formatting/#indent_use_margin).
```
